app/src/main/res/layout/view_video_item.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+                                   xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
`cmd + alt + l` ?

Reply:
![image](https://user-images.githubusercontent.com/42153507/98569101-84ab1c80-22ba-11eb-9b49-21acdf55aede.png)
Maybe because of ItelliJ IDEA

Reply:
may be, have you tried in AS?